### PR TITLE
integration-tests: add partial term to the find test

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

**Comment by niemeyer** [comment]
LGTM

**Comment by fgimenez** [comment]
retest this please

File: integration-tests/tests/snap_find_test.go [hunk 1]
```diff
@@ -34,14 +34,16 @@ type searchSuite struct {
 }
 
 func (s *searchSuite) TestSearchMustPrintMatch(c *check.C) {
-	searchOutput := cli.ExecCommand(c, "snap", "find", "hello-world")
-
 	// XXX: Summary is empty atm, waiting for store support
```
Comment (pedronis) [inline integration-tests/tests/snap_find_test.go]: summary is back now so we could look for that again I think

Comment (fgimenez) [inline integration-tests/tests/snap_find_test.go]: ok, IMO that should be done in another PR, the aim of this one is adding a partial name search to the test, what do you think?

Comment (pedronis) [inline integration-tests/tests/snap_find_test.go]: ok, let's merge this then
